po/ca.po

```diff
@@ -1852,7 +1851,7 @@ msgstr "_Visualitza"
 
 #: ../data/geany.glade.h:391
 msgid "Change _Font..."
-msgstr "Canvia el _tipus de lletra..."
+msgstr "Canvia la _lletra..."
```

Comment:
changing mnemonics isn't awesome, so 2 questions:
1. have you checked `l` isn't already used in that menu?
2. is the new mnemonic feeling more natural, or could/should you keep the `t` one? (as *lletra* has one, it would be doable without making it ugly and unexpected)

Not saying it's not a good change, just that it should possibly be done on purpose :)